ui-modules/app-inspector/app/components/entity-tree/entity-node.html

```diff
@@ -22,9 +22,12 @@
             <brooklyn-status-icon value="{{entity.serviceState}}" ng-if="entity.serviceState || entity.applicationId"></brooklyn-status-icon>
             <i class="fa fa-2x fa-external-link" ng-if="!entity.serviceState && !entity.applicationId"></i>
             <span>{{entity.name}}</span>
+            <span class="node-icon"><img ng-src="{{ iconUrl }}"/></span>
```

Comment:
Should use the `entity.iconUrl` instead of adding this into the scope, so the icon will be saved on the model which is propagated everywhere.

Reply:
nice idea but i think `$scope.entity` can be updated by the caller, so we'd need to watch for it to prevent it being cleared.  it isn't usually the icon which updates, it's things like status, so watching for that and reloading icon seems overkill.